src/views/ApartmentApp/components/StudentApplication/components/OffCampusList/index.js

```diff
@@ -4,8 +4,21 @@ import { Grid, Card, CardHeader, CardContent, List } from '@material-ui/core';
 import OffCampusListItem from './components/OffCampusListItem';
 import goStalk from 'services/goStalk';
 
-// Create a list of selection boxes to choosing which applicants are doing off campus programs.
-const OffCampusSection = ({ disabled, authentication, applicants, onOffCampusInputChange }) => {
+/**
+ * @typedef { import('services/housing').ApartmentApplicant } ApartmentApplicant
+ * @typedef { import('services/user').StudentProfileInfo } StudentProfileInfo
+ */
+
+/**
+ * Renders the list of selection boxes to choosing which applicants are doing off campus programs.
+ * @param {Object} props The React component props
+ * @param {Boolean} props.disabled Boolean to disable the interactive elements of this list item
+ * @param {*} props.authentication The user authentication
```

Comment:
`authentication` is a Boolean. Just leaving this note so you can update it everywhere.